kickstarter/android-oss
# NTV-336: Project Model

# 📲 What

- Migrated to kotlin
- Adopt @Parcelize 

# 🤔 Why
- Remove all references to AutoParcel library in Models, eventually we will be able to remove that dependency.

# 👀 See
- No user facing changes

|  |  |

# 📋 QA
- Open de app, navigate over all different categories on discovery screen.
- Open as many project pages as you can
- Open project from updates
- Check as many of your backed projects as you can
- Open messages, see your backings 
- Sign in as new user, add new payment method, and pledge to some project
- Browse project being logged out
- Log in as creator, view a pledge to your created project


# Story 📖

[NTV-336](https://kickstarter.atlassian.net/browse/NTV-336)
